rosidl_generator_py/resource/_msg_pkg_typesupport_entry_point.c.em

```diff
@@ -105,15 +105,15 @@ const rosidl_message_type_support_t *
 ROSIDL_GET_MSG_TYPE_SUPPORT(@(pkg_name), @(subfolder), @(spec.msg_name));
 
 int8_t
-_register_msg_type__@(type_name)(PyObject * pymodule)
+_register_msg_type__@(type_name)__@(subfolder)(PyObject * pymodule)
```

Comment:
@apojomovsky consider reversing the order of interpolated variables, as in `_register_msg_type__@(subfolder)_@(type_name)(PyObject * pymodule)`.

Reply:
Oh, you're right! Thanks for the suggestion, done!